api/peripheral_api/util/RFID-helpers.js

```diff
@@ -0,0 +1,122 @@
+const bcrypt = require('bcryptjs');
+const { RFID_BCRYPT_SALT } = require('../../config/config.json');
+const RFID = require('../models/RFID');
+const { OK, BAD_REQUEST, NOT_FOUND } =
+  require('../../util/constants').STATUS_CODES;
+const fs = require('fs');
+
+
+class RfidHelper {
+  constructor() {
+    this.addRfid = false;
+    this.name = null;
+  }
+
+  addingRfid() {
+    return this.addRfid;
+  }
+
+  keysExist() {
+    const filePaths = ['private.pem.key', 'cert.pem.crt', 'AmazonRootCA1.pem'];
+    for (const filePath of filePaths) {
+      try {
+        if (!fs.existsSync('../api/config/AWS-IOT/' + filePath)) {
+          return false;
+        }
+      } catch (e) {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  hashedByte(byte) {
+    return new Promise((resolve, reject) => {
+      bcrypt.genSalt(10, function(error, salt) {
+        if (error) {
+          resolve(null);
+        }
+        bcrypt.hash(byte, RFID_BCRYPT_SALT, function(hashError, hash) {
+          if (hashError) {
+            resolve(null);
+          }
+          resolve(hash);
+        });
+      });
+    });
+  }
+
+  getCards() {
```

Comment:
see my above comments, think you can write a function comment yourself? hint: the return value for this would be `Promise<Array<Object>>`.